src/web/components/ClickToView.tsx

```diff
@@ -121,17 +121,19 @@ export const ClickToView = ({ children, width, height, onAccept }: Props) => {
 					<Inner>
 						<Top>
 							<Headline width={width}>
-								The colourful beak is very large
+								Allow content provided by a third party?
 							</Headline>
 							<Body width={width}>
-								Quaerat quaerat ex nihil autem consequatur.
-								Velit rerum at ad dignissimos aut excepturi
-								ratione excepturi. Quaerat ipsam natus totam et
-								aut distinctio eaque voluptatem.
-								<Link href="https://theguardian.com">
-									{' '}
-									Quaerat ipsam
-								</Link>
+								<p>
+									This article includes content hosted on
+									other.com. We ask for your permission before
```

Comment:
One thing I was wondering in our last meeting was where we'd get `other.com` from. Perhaps it'd make sense to be in the CAPI response as `trackingDomain` or something?

Reply:
`other.com`_should_ eventually be replaced by a prop sent in from CAPI but we might need some fallback copy